Inc_Heated_Cylinders/flow_cylinder.cfg

```diff
@@ -0,0 +1,261 @@
+KT_CONSTANT= 0.0257
+%
+% Laminar Prandtl number (0.72 (air), only for CONSTANT_PRANDTL)
+PRANDTL_LAM= 0.72
+%
+% Turbulent Prandtl number (0.9 (air), only for CONSTANT_PRANDTL)
+PRANDTL_TURB= 0.90
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= GREEN_GAUSS
+%
+% Courant-Friedrichs-Lewy condition of the finest grid
+CFL_NUMBER= 10.0
+%
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= NO
+%
+% Parameters of the adaptive CFL number (factor down, factor up, CFL min value,
+%                                        CFL max value )
+CFL_ADAPT_PARAM= ( 1.5, 0.5, 10.0, 10000.0 )
+%
+% Runge-Kutta alpha coefficients
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+% Linear solver or smoother for implicit formulations (BCGSTAB, FGMRES, SMOOTHER_JACOBI,
+%                                                      SMOOTHER_ILU, SMOOTHER_LUSGS,
+%                                                      SMOOTHER_LINELET)
+LINEAR_SOLVER= FGMRES
+%
+% Preconditioner of the Krylov linear solver (ILU, LU_SGS, LINELET, JACOBI)
+LINEAR_SOLVER_PREC= ILU
+%
+% Linael solver ILU preconditioner fill-in level (0 by default)
+LINEAR_SOLVER_ILU_FILL_IN= 0
+%
+% Minimum error of the linear solver for implicit formulations
+LINEAR_SOLVER_ERROR= 1E-15
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 5
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+% Convective numerical method (JST, LAX-FRIEDRICH, CUSP, ROE, AUSM, HLLC,
+%                              TURKEL_PREC, MSW)
+CONV_NUM_METHOD_FLOW= FDS
+%
+% Monotonic Upwind Scheme for Conservation Laws (TVD) in the flow equations.
+%           Required for 2nd order upwind schemes (NO, YES)
+MUSCL_FLOW= YES
+%
+% Slope limiter (NONE, VENKATAKRISHNAN, VENKATAKRISHNAN_WANG,
+%                BARTH_JESPERSEN, VAN_ALBADA_EDGE)
+SLOPE_LIMITER_FLOW= NONE
+%
+% Time discretization (RUNGE-KUTTA_EXPLICIT, EULER_IMPLICIT, EULER_EXPLICIT)
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+
+% --------------------------- CONVERGENCE PARAMETERS --------------------------%
+%
+% Min value of the residual (log10 of the residual)
+CONV_RESIDUAL_MINVAL= -19
+%
+% Start convergence criteria at iteration number
+CONV_STARTITER= 10
+%
+% Number of elements to apply the criteria
+CONV_CAUCHY_ELEMS= 100
+%
+% Epsilon to control the series convergence
+CONV_CAUCHY_EPS= 1E-6
+
+% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
+%
+% Restart flow input file
+SOLUTION_FILENAME= solution_flow.dat
+%
+% Restart adjoint input file
+SOLUTION_ADJ_FILENAME= solution_adj.dat
+%
+% Output file format (TECPLOT, TECPLOT_BINARY, PARAVIEW,
+%                     FIELDVIEW, FIELDVIEW_BINARY)
+OUTPUT_FORMAT= TECPLOT
```

Comment:
Please change to `OUTPUT_FILES= ( RESTART_ASCII, PARAVIEW, SURFACE_PARAVIEW)` or to the output you prefer... to make it easier you might as well erase it in all the zone configs and just put one in the master

Reply:
Yes, ok.